apps/passport-server/src/services/multiProcessService.ts

```diff
@@ -62,6 +110,10 @@ export class MultiProcessService {
       },
       WORKER_MODULE_PATH
     );
+
+    this.jobs = {
+      verifySignaturePCD: new LruCache<Promise<boolean>>(100)
```

Comment:
could we bump this up to 1000 or maybe even 2000? for context the server we're using for production has 32 gb.